apps/web/src/lang/fr.json

```diff
@@ -9074,6 +9078,10 @@
     "defaultMessage": "À propos du Portail des talents autochtones",
     "description": "Talent Portal information heading"
   },
+  "lpx9/e": {
+    "defaultMessage": "Erreur: impossible de marquer les notifications comme lues.",
```

Comment:
```suggestion
    "defaultMessage": "Erreur : impossible de marquer les notifications comme lues.",
```

Reply:
Do you think it is worth to create a generic "Error" message and use the diving colon here?

Reply:
Oui! Also, found another French colon with the leading space.

## Screenshot
<img width="671" alt="Screen Shot 2024-06-18 at 08 25 50" src="https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/fa0b5f58-6b87-4947-acad-00b3c12a41c3">

Reply:
Okay, thanks :+1: Fixed in [fix translations](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10686/commits/ddc9e2cb2c865c52afb823dbf8a18dd05ff9c318)

I noticed we do this pattern a lot of "Error: Some specific message". Maybe we should update those all for consistency after this gets merged? :thinking: